extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/AtomicConstraintMapper.java

```diff
@@ -103,4 +105,16 @@ private AtomicConstraint buildAtomicConstraint(UiPolicyConstraint constraint) {
                 .rightExpression(new LiteralExpression(right))
                 .build();
     }
+
+    public AtomicConstraint buildAtomicConstraint(AtomicConstraintDto atomicConstraint) {
+        var left = atomicConstraint.getLeftExpression();
+        var operator = operatorMapper.getOperator(atomicConstraint.getOperator());
+        var right = atomicConstraint.getRightExpression();
+
+        return AtomicConstraint.Builder.newInstance()
+                .leftExpression(new LiteralExpression(left))
```

Comment:
Comment / Question: Left expression is an `Object`, right? This is a prime example of something that will have to be tested in E2E Tests, as we will only know very far "down the line" X whether the "policies" actually work.

Reply:
Both left and right expressions are of type `String` in the `AtomicConstraintDto` for now. I have not seen more complex `Expressions` to this point.